src/ConfigEditor/Auth/Auth.tsx

```diff
@@ -11,6 +11,7 @@ export type Props = {
   selectedMethod: AuthMethod | CustomMethodId;
   mostCommonMethod?: AuthMethod | CustomMethodId;
   visibleMethods?: Array<AuthMethod | CustomMethodId>;
+  extendedDefaultOptions?: Partial<Record<AuthMethod, DefaultAuthMethod>>;
```

Comment:
I'd suggest to rename this to `defaultOptionsOverrides` or something like this to make it clearer.